tests/MessagesTest.php

```diff
@@ -105,4 +126,56 @@ public function it_should_throw_a_missing_webhook_exception_when_client_didnt_re
         // When
         $slack->postMessage($message);
     }
+
+    /** @test */
+    public function it_should_return_the_name_of_the_channel()
```

Comment:
This isn't really a message test, is it? More like an AccessTokenTest or a SlackProviderTest?

I'd set up a test that tests the Provider, otherwise you'll end up testing all the tiny parts without knowing the whole thing works.